sql/4_materialized_views.sql

```diff
@@ -34,7 +34,7 @@ CREATE MATERIALIZED VIEW v_country_info AS
   FROM t_entrance e
   LEFT JOIN t_cave c ON e.id_cave = c.id AND c.is_deleted = false
   LEFT JOIN t_name n ON n.id_cave = c.id AND n.is_main = true
-  LEFT JOIN t_massif m ON ST_Contains(ST_SetSRID(m.geog_polygon::geometry, 4326), ST_SetSRID(ST_MakePoint(e.longitude, e.latitude), 4326))
+  LEFT JOIN t_massif m ON e.point_geom && m.geog_polygon AND ST_Contains(m.geog_polygon::geometry, e.point_geom)
```

Comment:
This is great to have this in here but it is also important to have a separate SQL script (prefixed with a sequence number and the date) so it can be run in isolation to upgrade existing databases like production.